app/models/runtime/space_application_supporter.rb

```diff
@@ -0,0 +1,9 @@
+module VCAP::CloudController
+  class SpaceApplicationSupporter < Sequel::Model(:spaces_application_supporters)
+    include SpaceRoleMixin
+
+    def type
+      @type ||= RoleTypes::SPACE_APPLICATION_SUPPORTER
```

Comment:
I know this was pattern matched from existing roles, but I don't think there is a benefit to saving a constant to an instance variable like this, is there?